src/main/java/jenkins/plugins/logstash/LogstashInstallation.java

```diff
@@ -95,13 +95,9 @@ public String getDisplayName() {
     }
 
     /**
-     * @see IndexerDaoFactory#getInstance(IndexerType, String, int, String, String)
+     * @see IndexerDaoFactory#getInstance(IndexerType, String, int, String, String, String)
      */
     public LogstashIndexerDao getIndexerDao() throws InstantiationException {
-      if (StringUtils.isEmpty(host)) {
-        return null;
-      }
-
```

Comment:
This was handling an edge case where the plugin has been installed but not yet configured:

1. Install the plugin, but don't edit the global configurations
2. Add the plugin to a job (either the post-build action or the buildwrapper)
3. Run the job

Here's what happens with the post-build action:

```
ERROR: Publisher jenkins.plugins.logstash.LogstashNotifier aborted due to exception
java.lang.NullPointerException
	at jenkins.plugins.logstash.LogstashInstallation$Descriptor.getIndexerDao(LogstashInstallation.java:101)
	at jenkins.plugins.logstash.LogstashNotifier.getDao(LogstashNotifier.java:109)
	at jenkins.plugins.logstash.LogstashNotifier.perform(LogstashNotifier.java:71)
	at hudson.tasks.BuildStepMonitor$3.perform(BuildStepMonitor.java:45)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:772)
	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:736)
	at hudson.model.Build$BuildExecution.post2(Build.java:183)
	at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:685)
	at hudson.model.Run.execute(Run.java:1757)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:88)
	at hudson.model.Executor.run(Executor.java:234)
```

With the buildwrapper it fails catastrophically (this is from the server log, not the build log):

```
SEVERE: #5 failed to build and we don't even have a listener
java.lang.NullPointerException
        at jenkins.plugins.logstash.LogstashInstallation$Descriptor.getIndexerDao(LogstashInstallation.java:101)
        at jenkins.plugins.logstash.LogstashBuildWrapper.getDao(LogstashBuildWrapper.java:95)
        at jenkins.plugins.logstash.LogstashBuildWrapper.decorateLogger(LogstashBuildWrapper.java:77)
        at hudson.model.Run.execute(Run.java:1711)
        at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
        at hudson.model.ResourceController.execute(ResourceController.java:88)
        at hudson.model.Executor.run(Executor.java:234)
```